### ISSUE TITLE
Notification Window is not opening in mobile after the latest update.

### ISSUE THREAD

**Issue by lianasamuel** [q2apro/q2apro-on-site-notifications#17]
Everything was working perfectly before the last update. But as I updated the plugin on my website, the notification window is not opening in mobile.
Please fix this ASAP.

**Comment by q2apro** [comment]
Have you updated today? 

I have added some CSS for mobiles, see https://github.com/q2apro/q2apro-on-site-notifications/commit/e232b13c59cd2be83be2724391f5d0967337b6ae

The new CSS fixes the window 100px on top of the screen.

**Comment by lianasamuel** [comment]
Yes I have updated the plugin today only. Means I have updated the plugin after your commits and from that point on notification window is not opening in mobiles.

**Comment by q2apro** [comment]
It opens but you cannot see it due to the CSS of your theme.

What is the URL of your website? Theme? And give me a guest login please.

**Comment by EtkaPerry** [comment]
hi
i have that problem and i use snowflat theme can you check it?
1.7.4

**Comment by raydingoz** [comment]
in plugin, style.css file snowflat*
.qam-account-items-wrapper .nfyWrap {
	***left:-437px;***  //when i delete this, it fixed.
	top:30px;
}

**Comment by O-TEE** [comment]
Am using Donut theme, it is not working for me.. please help!

**Comment by mugenishere** [comment]
I tested this on 1.8 and this fix is working fine here too!

**Comment by feeldattrend** [comment]
hello

I use 1.7 version of q2a  and  snow flat theme 

it is not working on mobile website. how can I fix it ? 

You wrote to disable snowflat global border box, but how ?

**Comment by mugenishere** [comment]
> 
> 
> hello
> 
> I use 1.7 version of q2a and snow flat theme
> 
> it is not working on mobile website. how can I fix it ?
> 
> You wrote to disable snowflat global border box, but how ?

Check out the comment above on this same page. Here is a link to it : https://github.com/q2apro/q2apro-on-site-notifications/issues/17#issuecomment-350562872

Go to style.css file. If you don't know where it is then use the windows find feature. I use everything search to locate files.

Open the file and comment out the line that he has mentioned in the comment.

**Comment by sonsuzus** [comment]
Please add css (like snowflat) for donut theme.
